docs/src/main/asciidoc/security-basic-authentication-concept.adoc

```diff
@@ -15,32 +19,46 @@ The header is specified as `Authorization: Basic <credentials>`, where credentia
 If the user name is `Alice` and the password is `secret`, the HTTP authorization header would be `Authorization: Basic QWxjZTpzZWNyZXQ=`, where `QWxjZTpzZWNyZXQ=` is a Base64 encoded representation of the `Alice:secret` string.
 ====
 
-The Basic Authentication mechanism does not provide confidentiality protection for the transmitted credentials.
+The Basic authentication mechanism does not provide confidentiality protection for the transmitted credentials.
 The credentials are merely encoded with Base64 when in transit and not encrypted or hashed in any way.
-Therefore, Basic Authentication is used with HTTPS to provide confidentiality.
+Therefore, Basic authentication is used with HTTPS to provide confidentiality.
 
-Basic Authentication is a well-specified, simple challenge and response scheme that all web browsers and most web servers understand.
-However, there are a few limitations associated with Basic Authentication, which include:
+Basic authentication is a well-specified, simple challenge and response scheme that all web browsers and most web servers understand.
 
-Credentials are sent as plain text::
-+
---
-Use HTTPS with Basic Authentication to avoid exposing the credentials.
-The risk of exposing credentials as plain text increases if a load balancer terminates HTTPS, as the request is forwarded to Quarkus over HTTP.
+== Limitations with using Basic authentication
+
```

Comment:
Should we have a rule about allowing uppercase B for basic when followed by authentication? common warning here.

Reply:
I'm on it... [we added one recently to the vale-at-red-hat repo, I will reuse that and other enhancements since the last rule update].